t/t7001-mv.sh

```diff
@@ -423,13 +423,13 @@ test_expect_success 'mv will error out on a modified .gitmodules file unless sta
 	entry="$(git ls-files --stage sub | cut -f 1)" &&
 	mkdir mod &&
 	test_must_fail git mv sub mod/sub 2>actual.err &&
-	test -s actual.err &&
-	test -e sub &&
+	test_file_not_empty actual.err &&
+	test_path_exists sub &&
 	git diff-files --quiet -- sub &&
 	git add .gitmodules &&
 	git mv sub mod/sub 2>actual.err &&
 	test_must_be_empty actual.err &&
-	! test -e sub &&
+	test_path_exists sub &&
```

Comment:
I think this should be `test_path_is_missing sub` instead, and I suspect that would fix [the test failure](https://github.com/git/git/actions/runs/3333459694/jobs/5515398258#step:4:1848), too.

Reply:
Thank you, let me retry but would that be for all of the 'test -e' subs or the '! test -e'?

Reply:
It would only be for `! test -e`, the `test -e` alone tests for the existence and the `!` inverses the condition.